public/lib/helpers/timeline_helper.js

```diff
@@ -26,6 +30,28 @@ define(function (require) {
       }
     };
 
+    TimelineHelper.prototype.changeTimezone  = function ($scope) {
```

Comment:
It's better if this method returns the moment option, which can then be picked up where necessary. So no need for the `$scope` argument.